website/docs/reference/resource-properties/concurrent_batches.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "concurrent_batches"
+resource_types: [models]
+datatype: model_name
+description: "concurrent_batches - Read this in-depth guide to learn about concurrent_batches in dbt."
+---
+
+<Tabs>
+<TabItem value="Project file">
+
+
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  +concurrent_batches: True
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="sql file">
+
+<File name='models/my_model.sql'>
+
+```sql
+{{
+  config(
+    materialized='incremental',
+    concurrent_batches=True,
+    incremental_strategy='microbatch'
+        ...
+  )
+}}
+select ...
+```
+
+</File>
+
+</TabItem>
+</Tabs>
+
+## Definition
+
+`concurrent_batches` is an override which allows users to decide whether or not they want to run their batches in parallel or sequentially (one at a time).
+
+## [`{{ this }}`](/reference/dbt-jinja-functions/this)
```

Comment:
i'm not sure i get the flow of this section here? if we're adding it here, why aren't we adding the full 'how parallel execution works' section in the microbach page?